packages/miraclecast.rb

```diff
@@ -32,8 +32,8 @@ class Miraclecast < Package
   depends_on 'gcc' # R
   depends_on 'glibc' # R
   depends_on 'glib' # R
-  depends_on 'llvm' # L
   depends_on 'gstreamer' # R
+  depwnds_on 'llvm' # R
```

Comment:
Typo. That "w" isn't supposed to be there: "dep**_w_**nds_on 'llvm'"

Reply:
Thanks. Fixed.